numba/tests/test_np_functions.py

```diff
@@ -1066,20 +1093,139 @@ def _triangular_matrix_exceptions(self, pyfunc):
             cfunc(a, k=1.5)
         assert "k must be an integer" in str(raises.exception)
 
+    def _triangular_indices_tests_n(self, pyfunc):
+        cfunc = jit(nopython=True)(pyfunc)
+
+        for n in range(10):
+            expected = pyfunc(n)
+            got = cfunc(n)
+            self.assertEqual(type(expected), type(got))
+            self.assertEqual(len(expected), len(got))
+            for e, g in zip(expected, got):
+                np.testing.assert_array_equal(e, g)
+
+    def _triangular_indices_tests_n_k(self, pyfunc):
+        cfunc = jit(nopython=True)(pyfunc)
+
+        for n in range(10):
+            for k in range(-n - 1, n + 2):
+                expected = pyfunc(n, k)
+                got = cfunc(n, k)
+                self.assertEqual(type(expected), type(got))
+                self.assertEqual(len(expected), len(got))
+                for e, g in zip(expected, got):
+                    np.testing.assert_array_equal(e, g)
+
+    def _triangular_indices_tests_n_k_m(self, pyfunc):
+        cfunc = jit(nopython=True)(pyfunc)
+
+        for n in range(10):
+            for m in range(2 * n):
+                for k in range(-n - 1, n + 2):
+                    expected = pyfunc(n, k, m)
+                    got = cfunc(n, k, m)
+                    self.assertEqual(type(expected), type(got))
+                    self.assertEqual(len(expected), len(got))
+                    for e, g in zip(expected, got):
+                        np.testing.assert_array_equal(e, g)
+
```

Comment:
Once you have the `n`  and `m`  versions above, you'll probably want to add an additional test setup here.

Reply:
Done